### DISCUSSION TITLE
HBO Max

### DISCUSSION THREAD

**Discussion by mastercho** [Purfview/IMDb-Scout-Mod#94] [General]
Did anyone managed to add HBO Max in IMBD Scout. Seems having encrypted headers which cant be done with Scout mod

**Comment by Purfview** [comment]
> encrypted headers

I dunno what that means. If you can share login then I could take a look at it.

**Reply by mastercho** [reply]
Hm i mean like Plex check if title exists in HBO MAX or not, i can give login if you leave any contact detail, discord, skype or email

**Comment by Purfview** [comment]
purfview at protonmail com

**Comment by mastercho** [comment]
After research found this https://greasyfork.org/en/discussions/requests/75196-peacock-and-hbo-max-subtitle-downloaders and for login into HBO Max need python libraries which is not supported by GreasyMonkey JS, so this idea just dead for now as cant be done.

**Comment by Purfview** [comment]
I don't see how some subtitle downloader is relevant to this script.
"Login" is not relevant too.

**Comment by Iceboy-6** [comment]
I partially add HBO Max catalog checker using TMDb watch providers section.
You can check here #110
